packages/contracts/src/diamond/Diamond.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {LibDiamond} from "./libraries/LibDiamond.sol";
+import {IDiamondCut} from "./interfaces/IDiamondCut.sol";
+
+contract Diamond {
+    constructor(address _contractOwner, address _diamondCutFacet) payable {
```

Comment:
In the constructor function of Diamond.sol, I think we need to put both addresses (OwnershipFacet and ManagerFacet you created). We can absolutely add new facet later but it doesn’t make sense to run twice for init deploy and upgrades.   so we can make it like this. 

`constructor (address _contractOwner, address[] _diamondCutFacets)`

 And then, I am not sure why the constructor should be payable function. The current code doesn’t execute any calldata and we don’t need to transfer any ETH.